sdks/java/extensions/sql/build.gradle

```diff
@@ -91,9 +91,14 @@ task generateFmppSources {
   }
 }
 
+// IntelliJ is happier when the directories are in the path, so we manually set this up
```

Comment:
Not just Intellij. Just say that we are making sure that the generated artifacts are placed in the appropriate package.

Reply:
Rephrased